subsys/usb/Kconfig

```diff
@@ -16,6 +17,24 @@ menuconfig USB_DEVICE_STACK
 
 if USB_DEVICE_STACK
 
+menuconfig USB_DEVICE_STACK_SETTINGS
+	bool
+	prompt "USB device stack settings"
+	default n
+	help
+	Allow to set USB device stack settings including Vendor ID
+	and Product ID.
+
+if USB_DEVICE_STACK_SETTINGS
```

Comment:
It makes no sense to make it optional, PID and VID must be explicitly configured.

Reply:
@jfischer-phytec-iot @pfl That is the idea first to introduce those parameters to make slow transition not affecting existing build sanity checks. The problem here is that we cannot assign default values here (unless you buy vendor id, in that case feel free to send a patch).